src/overrides/_spacing.scss

```diff
@@ -23,28 +23,30 @@ $spacing-directions: (
 // .govuk-\!-mt-r1 {
 //   margin-top: [whatever spacing point 1 is...]
 // }
+///
```

Comment:
Can the rest of this code block be updated to Sassdoc? (triple slash, use `@example`, document the parameters)?

Reply:
For sure, good shout @36degrees. Done now.